ivy_tests/test_ivy/test_functional/test_experimental/test_nn/test_layers.py

```diff
@@ -1268,3 +1268,68 @@ def test_rfftn(
         axes=axes,
         norm=norm,
     )
+
+
+@st.composite
+def x_and_irfftn(draw):
+    min_fft_points = 2
+    dtype = draw(helpers.get_dtypes("float"))
+    x_dim = draw(
+        helpers.get_shape(
+            min_dim_size=2, max_dim_size=100, min_num_dims=1, max_num_dims=4
+        )
+    )
+    x = draw(
+        helpers.array_values(
+            dtype=dtype[0],
+            shape=tuple(x_dim),
+            min_value=1e-10,
+            max_value=1e10,
+            large_abs_safety_factor=2.5,
+            small_abs_safety_factor=2.5,
+            safety_factor_scale="log",
+        )
+    )
+    axes = draw(
```

Comment:
you can use the `helpers.arrays_and_axes` helper instead of generating axes separately